website/components/PageWrapper/molecules/Header.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from "react";
+import routes from "common/routes";
+import Link from "next/link";
+import * as Styles from "../PageWrapper.styles";
+import { useRouter } from "next/router";
+import { SearchBar } from "./SearchBar";
+import { slide as Menu } from "react-burger-menu";
+import { checkMobileBreakPoint } from "common/util";
+
+export const Header: React.FC = () => {
+  const [isMobile, setIsMobile] = useState(true);
+  const router = useRouter();
+
+  const resizeListener = (): void => {
+    setIsMobile(checkMobileBreakPoint());
+  };
+
+  useEffect(() => {
+    // Resize on first render
+    resizeListener();
+
+    window.addEventListener("resize", resizeListener);
+    return () => window.removeEventListener("resize", resizeListener);
+  }, []);
```

Comment:
Just curious, but any reason for using a JS based mobile breakpoint vs CSS mobile breakpoint?

Reply:
Fair enough, top tier reason 😆